src/DssSpell.sol

```diff
@@ -19,28 +19,37 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-interface GemLike {
-    function transfer(address, uint256) external returns (bool);
-    function allowance(address, address) external view returns (uint256);
-    function approve(address, uint256) external returns (bool);
+interface VatLike {
+    function Line() external view returns (uint256);
+    function file(bytes32 what, uint256 data) external;
+    function ilks(bytes32 ilk) external view returns (uint256 Art, uint256 rate, uint256 spot, uint256 line, uint256 dust);
 }
 
-interface PauseLike {
-    function setDelay(uint256) external;
+interface VestLike {
+    function yank(uint256, uint256) external;
 }
 
-interface VestLike {
-    function restrict(uint256) external;
-    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
-    function yank(uint256) external;
+interface GemLike {
+    function transfer(address, uint256) external returns (bool);
```

Comment:
Not required. Param names are not included in the function selector.

Reply:
They are when you do `cast interface` and that's what the checklist says. I agree that it's pedantic and will discuss with Naz to find a middle ground that isn't just nitting every interface for 5 weeks (until that code rotates into archives). Not going to hold up the spell over this since we already saw this on Goerli and I agree, but just noting that it doesn't check the checklist item .